lib/hackerone/client/activity.rb

```diff
@@ -29,12 +29,28 @@ def attributes
       class BountyAwarded < Activity
         def bounty_amount
           formatted_bounty_amount = attributes.bounty_amount || "0"
-          formatted_bounty_amount.gsub(/[^\d]/, "").to_i
+          if ENV['HACKERONE_CLIENT_LENIENT_MODE']
```

Comment:
We should probably extract `'HACKERONE_CLIENT_LENIENT_MODE'` out to a constant. It may be used by other classes in the future so maybe it should live in https://github.com/oreoshake/hackerone-client/blob/master/lib/hackerone/client.rb